CONTRIBUTING.md

```diff
@@ -140,6 +140,11 @@ npm install --global proxay
 # or if you're using yarn
 yarn global add proxay
 ```
+When running the `./test.sh` script on iOS, it is also necessary to install `minio` using the following command:
```

Comment:
```suggestion
When running the `./test.sh` script on MacOS, it is also necessary to install `minio` using the following command:
```

Reply:
iOS? That is the mobile phone OS from Apple. Do you mean MacOS, their desktop operating system?

Reply:
And why is this an OS specific requirement? `minio` is an S3-API compatible object storage service. I believe we use that for testing S3 compatibility locally. Hence, you probably need that on Linux/(Windows) too if you want to run the tests, right?